IdentityCore/src/cache/MSIDMacTokenCache.m

```diff
@@ -278,6 +278,13 @@ - (BOOL)saveAppMetadata:(MSIDAppMetadataCacheItem *)item
 
 #pragma mark - Removal
 
+- (BOOL)removeTokensWithKey:(MSIDCacheKey *)key
+                    context:(id<MSIDRequestContext>)context
+                      error:(NSError **)error
+{
+    return [self removeItemsWithKey:key context:context error:error];
+}
+
```

Comment:
nit: let's call the other public method removeAccounts to distinguish and be specific?

Reply:
nit2: agreed

Reply:
Addressed. 
The other method will be used to remove both account and metadata. So I added `removeItemsWithAccountKey` and `removeItemsWithMetadataKey`, while making `removeItemsWithKey` as private :)